sky/tools/create_full_ios_framework.py

```diff
@@ -164,8 +164,25 @@ def create_framework(
 
 
 def zip_archive(dst):
+  entitlement_config = open(dst + '/entitlements.txt', 'w')
```

Comment:
use os.path join to join the paths and a with context to open the file.

Reply:
Good point, changed.